### DISCUSSION TITLE
Navigation arrow slides multiple slides per click

### DISCUSSION THREAD

**Discussion by nerdcel** [nolimits4web/swiper#6061] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/s/busy-aryabhata-dmgxkh?file=/src/index.js

### Bug description

If the option "slidesPerView" is used, clicking on a navigation arrow causes the slider to be moved by 2 slides instead of 1.
How can it be controlled that X slides are moved per click? slidesPerGroup does not seem to work.
If the option "loop" is set to true, only 1 slide moves per click. In this case, however, no loop should be used.

### Expected Behavior

Only 1 slide per click is moving, or slidesPerGroup has an effect on that behavior.

### Actual Behavior

If more than 1 slidePerView is set, 2 slides are sliding per click.

### Swiper version

8.3.2

### Platform/Target and Browser Versions

Chrome

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by elliottmangham** [comment]
I'm in need of this also, is it possible to make the arrow click move a specific number of slides at certain breakpoints? I.e. click on next moves 3 slides, but 1 on mobile?

**Comment by tgwow** [comment]
> I'm in need of this also, is it possible to make the arrow click move a specific number of slides at certain breakpoints? I.e. click on next moves 3 slides, but 1 on mobile?

Yes, see the `breakpoints` prop, you can set values to each breakpoint you wanna, eg:
```jsx
breakpoints={{
  440: {
    slidesPerView: 2,
    slidesPerGroup: 2,
    spaceBetween: 10,
  },
  640: {
    slidesPerView: 3,
    slidesPerGroup: 3,
    spaceBetween: 10,
  }
 }
....
```

**Reply by sajidsalman75** [reply]
Same here I am facing the same bug when I click on the next slide arrow. It slides 2 slides instead of 1 which is default. Explicitly defining it 1 is also not working.

**Comment by ecovaci-gd** [comment]
Hey @nerdcel, I'm facing the same problem, exactly as you described. 

Did you find any solutions for this?
